averbis/core/_rest_client.py

```diff
@@ -874,7 +979,28 @@ def create_and_run_process(
         :return: The created process
         """
         # noinspection PyProtectedMember
-        self.project.client._create_and_run_process(self, process_name, pipeline)
+        self.project.client._create_and_run_process(self, process_name, pipeline, Process.ProcessType.MACHINE)
+
+        return self.get_process(process_name)
+
+    @experimental_api
+    def create_process(
+            self,
+            process_name: str,
+            is_manual_annotation: bool = False) -> Process:
+        """
+        HIGHLY EXPERIMENTAL API - may soon change or disappear.
+
+        Creates a process without a pipeline (e.g. for later manual annotation or text analysis result import)
+        :param: is_manual_annotation the created process will be used for manual annotation
+        i.e. the underlying data structure will be initialized immediately and not on later text analysis result import
+        :return: The created process
+        """
+        process_type = Process.ProcessType.IMPORTED
```

Comment:
IMHO instead of saying `IMPORTED` or `MANUAL`, we should have a boolean option `initializeEmptyResults`. The potential implications of setting the type directly are too broad.